client/src/pages/Checkout/Checkout.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react';
+import { Box, FormControl, OutlinedInput, Button, MenuItem, Select } from '@material-ui/core';
+
+import Navigation from '../Navigation/Navigation';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import useStyles from './useStyles';
+
+interface CardInfo {
+  cardNumber: string;
+  expirationMonth: string;
+  expirationYear: string;
+  cvc: string;
+}
+
+const Checkout = (): JSX.Element => {
+  const classes = useStyles();
+  const [cardInfo, setCardInfo] = useState<CardInfo>({
+    cardNumber: '',
+    expirationMonth: '',
+    expirationYear: '',
+    cvc: '',
+  });
+
+  const months = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'];
+  const years = ['21', '22', '23', '24', '25', '26', '27', '28', '29', '30'];
```

Comment:
While the months is okay, the years is not practical. I don't want to have to maintain this list of years for expiration date. I would suggest either refactoring and just letting the user input the format `MM/YY` or how years a bit more automated. 

Truthfully, you could just look into letting them input `MM/YY` and mask the input and then do further logic to make sure the card isn't already expired or let Stripe handle that logic.

Reply:
Hey Ethan, I've decided to let the user input the MM/YY. Also, I only masked the CVC because the mock up doesn't mask the expiration date. However, if you want me to mask it, let me know! Lastly, I'm going to let Stripe handle that logic. Thanks for the feedback!